### PR TITLE
Eobs 360 doctor task assessment required

### PR DESCRIPTION
**Pull request by Gimpneek**
- Amended demo data so doctors are assigned to beds by default
- Added test to ensure doctor task is being triggered correctly

### CODE REVIEW

File: nh_ews/notifications.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ class nh_clinical_notification_medical_team(orm.Model):
     _name = 'nh.clinical.notification.medical_team'
     _inherit = ['nh.clinical.notification']
     _description = 'Inform Medical Team?'
-    _notifications = [{'model': 'doctor_assessment', 'groups': ['nurse']}]
+    _notifications = [{'model': 'doctor_assessment', 'groups': ['doctor']}]
```
Comment (Nayira) [inline nh_ews/notifications.py]: Might be getting confused here, but the 'Inform Medical Team' task is supposed to be seen (and completed) by the 'Nurse' group, not the 'Doctor' group. Why are we changing this one?

Comment (Gimpneek) [inline nh_ews/notifications.py]: Groups in notifications are the user group to trigger the notification for so as the task 'Assessment Required' is triggered for doctors on an inform medical team task being completed it should have been doctor and not nurse.

**Comment by Nayira** [comment]
Code review passed (in the absence of Dan) - :shipit:

**Comment by Nayira** [comment]
Feature branch tests for EOBS-360 have passed :shipit:

**Pull request merged**